src/com/spikes2212/gui/DashBoardController.java

```diff
@@ -0,0 +1,92 @@
+package com.spikes2212.gui;
```

Comment:
What does this file have to do with anything? please revert the commit that added this, it should have it's own branch